project-base/tests/ShopBundle/Database/EntityExtension/EntityExtensionTest.php

```diff
@@ -69,6 +71,7 @@ public function setUp()
             OrderTransport::class => ExtendedOrderTransport::class,
             Brand::class => ExtendedBrand::class,
             Payment::class => ExtendedPayment::class,
+            Order::class => ExtendedOrder::class,
```

Comment:
Same.

Reply:
it is in the OrderItem of Framework Model